universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java

```diff
@@ -51,6 +51,13 @@ public Content render(HtmlBundle bundle) {
   }
 
   public Content renderWithNav(Http.Request request, Messages messages, HtmlBundle bundle) {
+    // TODO: This will set the html lang attribute to the requested language when we actually want
```

Comment:
question: Why is this a TODO?

Reply:
I didn't want to add any additional parameters to `renderWithNav` or throughout the `applicantLayout' and this was the best I could do without doing that.